### DISCUSSION TITLE
Vuetify treeview does not compile

### DISCUSSION THREAD

**Discussion by karolina-ilgine** [directus/directus#11154] [Archive]
I am trying to use v-treeview component in a custom layout, but it seems that it is not building properly, even though other vuetify components work well:
![image](https://user-images.githubusercontent.com/89016232/150135503-3404e9d7-48ef-4c03-8617-d18c7628a65a.png)

Do you have any ideas what is wrong?

**Comment by nickrum** [comment]
Just to confirm, are you actually importing Vuetify Components inside your extension or are you using the `v-` prefixed components registered by the Directus App?

In case you are doing the latter: At the moment, we do not provide a `v-treeview` component.

**Reply by karolina-ilgine** [reply]
Unfortunately, the latter. Thank you for a quick response!

**Reply by karolina-ilgine** [reply]
One more question, do you have any examples of using imported Vuetify components?

**Reply by nickrum** [reply]
That might be tricky as we are using Vue 3 and the latest version of Vuetify doesn't yet support Vue 3. They do have an [alpha version](https://vuetifyjs.com/en/introduction/roadmap/#in-development), though. You might be able to directly import components from that version and localy register them inside your layout's root component.

**Reply by MyUserNameIsMy** [reply]
any updates regarding this?
